turbopack/crates/turbopack-ecmascript/src/analyzer/imports.rs

```diff
@@ -374,15 +375,13 @@ impl Visit for Analyzer<'_> {
             self.data.imports.insert(local, (i, orig_sym));
         }
 
-        if import.specifiers.is_empty() {
-            if let Some(internal_symbol) = internal_symbol {
-                self.ensure_reference(
-                    import.span,
-                    import.src.value.clone(),
-                    internal_symbol,
-                    annotations,
-                );
-            }
```

Comment:
I think that change is necessary. `ensure_reference` is already called for every specifier, so it doesn't need to be called here again.